glue/viewers/matplotlib/state.py

```diff
@@ -273,7 +273,7 @@ class MatplotlibLayerState(LayerState):
 
     def __init__(self, viewer_state=None, **kwargs):
 
-        super(MatplotlibLayerState, self).__init__(viewer_state=viewer_state, **kwargs)
+        super(MatplotlibLayerState, self).__init__(viewer_state=viewer_state, preferred_cmap=None, **kwargs)
```

Comment:
Is this needed?

Reply:
Yeah, you are right. I checked and indeed this code is not needed.